dim.sh

```diff
@@ -15,21 +15,22 @@ curl -L -O $URL
 unzip $ZIP_NAME
 cd $FILE_NAME
 
-# setup.sh is DOS encoded
-tr -d '\015' <setup.sh >setup2.sh 
+# setup.sh is DOS-encoded
+tr -d '\015' < setup.sh > setup2.sh
 mv setup2.sh setup.sh
 
-# compile
-export OS=Linux
-source setup.sh
-gmake realclean
-gmake X64=yes
+# Build (needs special environment)
+( export OS=Linux
+  source setup.sh
+  gmake realclean
+  gmake X64=yes )
```

Comment:
Those parentheses are like the Fight Club. Whatever happens there (environment-wise), stays there :-)

Jokes aside: it's an inner shell, so whatever `export`, etc. will not be available outside of them.